src/plugins/drawing/model/drawing-content.ts

```diff
@@ -124,8 +124,10 @@ export const DrawingContentModel = TileContentModel
       const {name, ...loggedChangeProps} = call;
       // TODO: logTileChange includes an explicit DrawingToolLogEvent
       // this isn't a good pattern to support generic plugins logging events
-      Logger.logTileChange(LogEventName.DRAWING_TOOL_CHANGE, name,
-        loggedChangeProps, self.metadata?.id ?? "");
+      if (name !== "setDisabledFeatures"){
```

Comment:
Not sure if this is the best solution, but it works and prevents any drawing tiles from calling onTileAction which logs DRAWING_TOOL_CHANGE to the logger upon opening up a document with drawing tiles.